.docker/plugins/azureud/.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.6
```

Comment:
Hi @codenrhoden,

Why create a plug-in-specific `.Dockerfile` for this plug-in? The only difference is the version of Alpine. Is 3.6 required? If so, why not bump Alpine's version for the root [`Dockerfile`](https://github.com/codedellemc/rexray/blob/master/.docker/plugins/Dockerfile)?

Reply:
Your answer is in the description of the PR. This plugin requires `lsscsi`.

I do plan to bump all plugins to Alpine 3.6 in a separate commit/PR.

Reply:
Hi @codenrhoden,

You expected me to read the description? :)

(doh! sorry)